webapp/components/Select/ActionRadiusSelect.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <select
+    class="select ds-input appearance--auto"
+    name="actionRadius"
+    :value="value"
+    @change="onActionRadiusChange"
+  >
+    <option v-for="actionRadius in actionRadiusOptions" :key="actionRadius" :value="actionRadius">
+      {{ $t(`group.actionRadii.${actionRadius}`) }}
+    </option>
+  </select>
+</template>
+
+<script>
+export default {
+  name: 'ActionRadiusSelect',
+  props: {
+    value: {
+      required: true,
+    },
+  },
+  data() {
+    return {
+      actionRadiusOptions: ['regional', 'national', 'continental', 'global'],
```

Comment:
I assume it would be wise to remove interplanetary from the enum - if you could do this in this PR that be great - but maybe someone has some info about this.